lib/payment_processor/go_cardless/populator.rb

```diff
@@ -26,7 +26,11 @@ def subscription_params
           {
             name: "donation",
             interval_unit: "monthly"
-          }
+          }.tap do |params|
+            if bacs?
+              params[:start_date] = subscription_start_date
```

Comment:
Is this in the format that expected by the API? I think you might need to call `.to_s` on the date.